[PDE-4035] chore(cli, core, schema) release 14.0.1 (#649)
zapier/zapier-platform: CHANGELOG.md

## 14.0.1

_released `2023-05-22`_

### cli

> - :scroll: Fix missing `key` property in the `.zapierapprc` file after running the `zapier convert` command

Reviewer: We typically provide a link to the PR for each item in the CHANGELOG.
```suggestion
- :scroll: Fix missing `key` property in the `.zapierapprc` file after running the `zapier convert` command [#640](https://github.com/zapier/zapier-platform/pull/640)
```

Author: Oops, that's right! I've been playing around with auto-generating these and at this point I've forgotten how they are supposed to look like 😵 Will add this 🙏🏻

## 14.0.1

_released `2023-05-22`_

### cli

> - :scroll: Fix duplicate authentication field keys being generated by the `zapier convert` command ([#639](https://github.com/zapier/zapier-platform/pull/639))
> - :scroll: Fix missing `key` property in the `.zapierapprc` file after running the `zapier convert` command ([#640](https://github.com/zapier/zapier-platform/pull/640))

Reviewer 2: These two are bug fixes 🐛 

```suggestion
- :bug: Fix duplicate authentication field keys being generated by the `zapier convert` command ([#639](https://github.com/zapier/zapier-platform/pull/639))
- :bug: Fix missing `key` property in the `.zapierapprc` file after running the `zapier convert` command ([#640](https://github.com/zapier/zapier-platform/pull/640))
```

- :scroll: Improve docs related to expected behaviors for stale authentication ([#647](https://github.com/zapier/zapier-platform/pull/647))

### core

- None!

### schema

> - :exclamation: Deprecate the `important` key of the `BasicDisplaySchema`. This property will be removed in the next major version release. ([#644](https://github.com/zapier/zapier-platform/pull/644))

Reviewer 2: This is more like a document update, so I would stick with the rule and use 📜. But I understand we probably want to make it more visible, so maybe we can do ❗ in the sentence?

```suggestion
- :scroll: Deprecate the `important` key of the `BasicDisplaySchema`. This property will be removed and hence a breaking change ❗ in the next major version release. ([#644](https://github.com/zapier/zapier-platform/pull/644))
```
